src/FocusApp/MauiProgram.cs

```diff
@@ -23,6 +24,8 @@ public static MauiApp CreateMauiApp()
                     fonts.AddFont("Font-Awesome-6-Free-Regular.otf", nameof(LineArtIcons));
                 });
 
+            builder.Services.AddSingleton<APIService>();
```

Comment:
I think refit's dependency injection might handle the singleton situation. Was this necessary to get the code to work or would we be able to remove it?